### PR TITLE
Show group creation and member count in group detail view

### LINKED ISSUE #255: Group details improvements
- Add group creator back to group members for groups where current user is the creator
- List number of members in group and group member limit
- List group creation date

### PR DESCRIPTION
**Pull request by sillych**
Similar to the android app group view (Group creation date value available after the next app update)

### CODE REVIEW

**Comment by sillych** [comment]
group-1

File: src/threema.d.ts [hunk 1]
```diff
@@ -245,6 +245,7 @@ declare namespace threema {
         members: string[];
         administrator: string;
         access: GroupReceiverAccess;
+        createdAt: string;
```
Comment (dbrgn) [inline src/threema.d.ts]: This should be optional (`createdAt?`), since older apps don't send it, right?

Comment (sillych) [reply src/threema.d.ts]: yes!

**Comment by sillych** [comment]
Fix #255

**Comment by rugk** [comment]
In the screenshot, I cannot see the group creation date requested in https://github.com/threema-ch/threema-web/issues/255.

**Comment by dbrgn** [comment]
@rugk because the group was created the same day.

**Comment by sillych** [comment]
@rugk he same behaviour like threema on android.

**Pull request merged**